ocf_data_sampler/load/nwp/providers/ecmwf.py

```diff
@@ -27,10 +26,14 @@ def open_ifs(zarr_path: Path | str | list[Path] | list[str]) -> xr.DataArray:
     ds = ds.rename(
         {
             "init_time": "init_time_utc",
-            "variable": "channel",
         }
     )
 
+    # LEGACY SUPPORT
+    # rename variable to channel if it exists
+    if "variable" in ds:
+        ds = ds.rename({"variable": "channel"})
+
```

Comment:
big question, should we remove stuff like this? I'm not totally sure how easy it is to sort the data out that we load

Reply:
I dont mind code like this, fairly easy to read, and covers some edge cases of data

Reply:
I think as long as we have archives in use that nee this, we are fine keeping it in. It might not be the cleanest but the root issue is not in the code so that's fine. I'm more concerned that there is ecmwf archives we use that will not be opened by this.